sentry-android-core/src/main/java/io/sentry/android/core/NdkIntegration.java

```diff
@@ -46,4 +52,10 @@ public final void register(IHub hub, SentryOptions options) {
       options.setEnableNdk(false);
     }
```

Comment:
L:
```suggestion
    }
    else {
        options.getLogger().log(DEBUG, "NDK won't be initialized"):
    }
```

Reply:
this is already implicit by:

`options.getLogger().log(SentryLevel.DEBUG, "NdkIntegration enabled: %s", enabled);`